### DISCUSSION TITLE
Why the region us-east-1 is required?

### DISCUSSION THREAD

**Discussion by rclemente-mrge** [aws-solutions-library-samples/cloud-intelligence-dashboards-framework#951] [Q&A]
Hello

We are currently trying to build a CUDOS dashboard and we wondering why your modules requires the provider to be us-east-1

https://github.com/aws-samples/aws-cudos-framework-deployment/blob/main/terraform-modules/cur-setup-destination/versions.tf#L8

Thanks

**Comment by iakov-gan** [comment]
@sean-nixon Can you check if it is a hard requitement for TF ? CFN do not have this dependency and can be installed in any region. 

cur-setup-source surely must be in us-east-1. 
https://github.com/aws-samples/aws-cudos-framework-deployment/blob/main/terraform-modules/cur-setup-source/main.tf#L253 

But i am not sure if it is required in destination or TF module can do cross region replicaiton and allow to set up in ManagementAccount/us-east-1 and replicate to SourceAccount/region-x. Surely this implies x region daily replication charges that we do not have with CFN.

**Comment by sean-nixon** [comment]
CUR must be created in us-east-1 for the standard `aws` partition. CFN is using a Lambda function to do this regardless of the CFN region but with Terraform we can use different regions with different provider configurations. For this reason, we require at least one provider configuration to be passed to the module that is configured with us-east-1 or the Terraform deployment will fail. All other resources in both the source and destination module can be created in any supported region using the default `aws` provider passed explicitly or implicitly to the module.
